bucket/zulumc.json

```diff
@@ -1,28 +1,28 @@
 {
-    "homepage": "https://www.azul.com/products/zulu-mission-control",
     "version": "8.1.0.42",
-    "description": "Zulu Mission Control",
+    "description": "A build of JDK Mission Control, an open source Java runtime profiling and monitoring utility",
+    "homepage": "https://www.azul.com/products/components/zulu-mission-control/",
     "license": "UPL-1.0",
     "architecture": {
         "64bit": {
             "url": "https://cdn.azul.com/zmc/bin/zmc8.1.0.42-ca-win_x64.zip",
             "hash": "a430e6b797c805d4e0e40bd66fbff79ff77d3de4b5394a75c6596c09abcb7143",
-            "extract_dir": "zmc8.1.0.42-ca-win_x64"
+            "extract_dir": "zmc8.1.0.42-ca-win_x64\\Azul Mission Control"
         }
     },
+    "bin": "zmc.exe",
     "shortcuts": [
         [
             "zmc.exe",
             "Zulu Mission Control"
         ]
     ],
-    "persist": "configuration",
-    "checkver": "zmc([\\d.]+)(?<eaca>-EA|-ca)?-win",
+    "checkver": "zmc([\\d.]+)-ca-win",
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "https://cdn.azul.com/zmc/bin/zmc$version$matchEaca-win_x64.zip",
-                "extract_dir": "zmc$version$matchEaca-win_x64"
+                "url": "http://cdn.azul.com/zmc/bin/zmc$version-ca-win_x64.zip",
```

Comment:
Do use https, as it is available.

Reply:
Changing to https.
There isn't going to be early access mc releases for foreseeable future and even if it is decided to bring back those. I think it should be a separate manifest.